cmake/BuildFlags.cmake

```diff
@@ -116,7 +116,11 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "MSVC"
   set(SYCL_OFFLINE_COMPILER_CG_OPTIONS "${SYCL_OFFLINE_COMPILER_CG_OPTIONS} -cl-fp32-correctly-rounded-divide-sqrt")
   set(SYCL_OFFLINE_COMPILER_CG_OPTIONS "-options '${SYCL_OFFLINE_COMPILER_CG_OPTIONS}'")
 
-  set(AOT_TARGETS "pvc,xe-lpg,ats-m150")
+  if(WIN32)
+    set(AOT_TARGETS "ats-m150,lnl-m,mtl-u,mtl-h")
+  else()
+    set(AOT_TARGETS "pvc,xe-lpg,ats-m150")
```

Comment:
Since it kind of depends on the driver implementation, should we replace xe-lpg with mlt-u and mtl-h? It is not critical for PT 2.5